packages/vkui/src/components/Radio/Radio.tsx

```diff
@@ -85,6 +85,7 @@ export const Radio = ({
         {...restProps}
         Component="input"
         type="radio"
+        role="radio"
```

Comment:
А нам точно надо выставлять `role` у инпута с типом "radio"?
Я думаю, что это [избыточно](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role).

> Note: The first rule of ARIA is if a native HTML element or attribute has the semantics and behavior you require, use it instead of re-purposing an element and adding ARIA. Instead use the native [HTML <input type="radio">](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio) (with an associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label), which natively provides all the functionality required:

В дереве доступность вполне чётко и так прописано `radio`.
<img width="940" alt="Screenshot 2024-06-27 at 20 26 39" src="https://github.com/VKCOM/VKUI/assets/5443359/349b47f7-1d89-46ef-997c-a3a09f4c9971">

Reply:
Согласен, скорее всего избыточно, убрал